CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+## Rules
+
+Please observe the following ground-rules when making contributions:
+
+- **All** contributions must be made by opening a pull request into `main`
+- Please add a description to the pull request describing _what_ features you're
+  implementing or issues you are solving and _how_
+- Commit summaries **must** use the imperative, i.e. "Fix typo" instead of
+  "Fixed typo" or "Fixes typo"
+- Avoid force pushes after requesting a review
+- A pull request may only be merged when it has received at least one approval
+  from a member of [zeitgeistpm](https://github.com/zeitgeistpm) and GitHub
+  Actions have passed successfully
+- Please use _squash and merge_ to merge pull requests; feel free to remove
+  trivial items like "Fix typo", etc. from the commit summary
```

Comment:
Do we need to mention the rules for git labels here?
For me at least it was very confusing first (What is `s`, `p`, `t`? When to use what ?).

Reply:
Who should resolve the conversation in the review comments?

Reply:
Who manages the milestones?

Reply:
When fixing an issue, use `Development -> Create Branch` to link it with the PR.

Reply:
Who should close an issue or PR?

Reply:
I think milestone management is restricted to team member. I don't think we need detailed rules for that, but feel free to make a suggestion.

Reply:
Oh ok, it was just no clear to me if it's about the rules inside the Zeitgeist Team too. So it's primarily for outsider contributions. Then it's okay.